warehouse/packaging/models.py

```diff
@@ -424,6 +450,13 @@ def __table_args__(cls):  # noqa
         passive_deletes=True,
     )
 
+    vulnerabilities = orm.relationship(
+        Vulnerability,
+        backref="releases",
+        secondary=lambda: release_vulnerabilities,
+        passive_deletes=True,
+    )
+
```

Comment:
Should probably define this on the `Vulnerability` model instead.

Reply:
Would we not want both?